src/utils/string/StringUtils.js

```diff
@@ -4,7 +4,7 @@ import * as React from 'react';
 import moment from 'moment';
 import reactStringReplace from 'react-string-replace';
 import _, {isArray} from 'lodash';
-import {Linking, Pressable, StyleSheet, Text} from 'react-native';
+import {Linking, StyleSheet, Text} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `Pressable` has been removed but it's not clear if this component is no longer used in the file. If it's still being used, this could cause a reference error.
```diff
- import {Linking, Pressable, StyleSheet, Text} from 'react-native';
+ import {Linking, StyleSheet, Text} from 'react-native';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->